src/test/java/com/exasol/bucketfs/AbstractBucketIT.java

```diff
@@ -11,25 +11,47 @@
 import com.exasol.config.BucketFsServiceConfiguration;
 import com.exasol.containers.ExasolContainer;
 
+/**
+ * Abstract class used for integration tests
+ */
 @Testcontainers
 public abstract class AbstractBucketIT {
```

Comment:
for test-code javadoc is not required...

Reply:
In places where it does not add any information I would suggest to remove it.

Reply:
Idk why but otherwise javadoc errors out.